components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -953,7 +958,8 @@ private AuthenticatedIdPData getAuthenticatedIdPData(AuthenticationContext conte
     }
 
     protected String getRedirectUrl(HttpServletRequest request, HttpServletResponse response, AuthenticationContext
-            context, String authenticatorNames, String showAuthFailureReason, String retryParam, String loginPage)
+            context, String authenticatorNames, String showAuthFailureReason, String
```

Comment:
we can't change the non-private method

Reply:
Fixed with 10807f5f92bd1a00b33004aed54ba9524f2d1994